.github/workflows/clt_nightly.yml

```diff
@@ -20,6 +20,9 @@ on:
       - 'test/clt-tests/indexer-latest-mysql-postgres/**'
       - 'test/clt-tests/core/test-alter-rename-nightly'
       - 'test/clt-tests/integrations/**'
+      - 'test/clt-tests/core/test-performance-for-queries-with-multiple-disk-chunks'
+      - 'test/clt-tests/integrations/**'
```

Comment:
duplicate

Reply:
Also it looks like these changes are not related with the task of the PR. Not a big deal, but don't forget to mention them separately in the commit then.

Reply:
Fixed